editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1993,6 +1998,24 @@ void CanvasItemEditor::_viewport_draw() {
 
 		Rect2 rect = canvas_item->get_item_rect();
 
+		if (drag != DRAG_NONE && drag != DRAG_PIVOT) {
```

Comment:
There are a couple style issues in this chunk: https://travis-ci.org/godotengine/godot/jobs/274700640